### DISCUSSION TITLE
build-with-quarto badge

### DISCUSSION THREAD

**Discussion by nichtich** [quarto-dev/quarto-cli#831] [Q&A] [answered]
Has anyone created a project badge yet? I am going to add this to repositories that use quarto:

![https://img.shields.io/badge/build_with-quarto-69b1e9](https://img.shields.io/badge/build_with-quarto-69b1e9)

**Comment by jjallaire** [comment]
No official project badge yet but that one looks good! Maybe "built with" instead of "build with" ?

**Reply by nichtich** [reply, marked as answer]
Ok via https://img.shields.io/badge/built_with-quarto-69b1e9:

![](https://img.shields.io/badge/built_with-quarto-69b1e9)
